diagnostic.md

```diff
@@ -35,5 +41,9 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{#each model as |phone|}}
+    {{my-contact/my-phone number=phone.number}}
+    {{/each}}
```

Comment:
Instead of `model` we need to access the object property `contact.phones`.  you don't need `contacts/`.  So it will look like this:
```
{{#each contact.phones as |phone|}}
{{my-phone number=phone.number}}
{{/each}}
```